app/services/classification/base.rb

```diff
@@ -160,6 +160,12 @@ def get_element_and_element_name(lookup_value)
       element_name = ''
       lookup_value.match.split(LookupKey::KEY_DELM).each do |match_key|
         lv_element, lv_element_name = match_key.split(LookupKey::EQ_DELM)
+        case lv_element
```

Comment:
I'm fine with not changing this but that means you would look at the additional information for smart class parameters and see 'hostgroup' and in global parameters 'host group'.